### DISCUSSION TITLE
How do I integrate sending notification via Viber or Telegram?

### DISCUSSION THREAD

**Discussion by nguyenbach0423** [Budibase/budibase#5458] [Help] [answered]
How do I integrate sending notification via Viber or Telegram? I don't know how to use integromat integration. Help me!!!

**Comment by melohagan** [comment, marked as answer]
Hey @nguyenbach0423 

It appears both have a REST API, which Budibase can interact with:

 - https://developers.viber.com/docs/api/rest-bot-api/
 - https://core.telegram.org/
 - https://docs.budibase.com/docs/rest

**Reply by nguyenbach0423** [reply]
thank you!!!
